web/src/settings_config.ts

```diff
@@ -628,7 +628,9 @@ export const realm_user_settings_defaults_labels = {
         defaultMessage: "Display availability to other users",
     }),
     realm_presence_enabled_parens_text: $t({defaultMessage: "invisible mode off"}),
-    realm_enter_sends: $t({defaultMessage: "Enter sends when composing a message"}),
+    realm_enter_send: new Handlebars.SafeString(
+        $t_html({defaultMessage: "<kbd>Enter</kbd> sends when composing a message"}),
+    ),
```

Comment:
There is no need for this since it can use the label from `preferences_settings_labels`.

Reply:
Updated.